modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/api/construction/ConstructionMetadata.scala

```diff
@@ -0,0 +1,21 @@
+package org.tessellation.rosetta.domain.api.construction
+
+import org.tessellation.ext.derevo.magnoliaCustomizable.snakeCaseConfiguration
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.transaction.{TransactionFee, TransactionReference, TransactionSalt}
+
+import derevo.cats.show
+import derevo.circe.magnolia.customizableDecoder
+import derevo.derive
+
+case object ConstructionMetadata {
```

Comment:
why there is a need for `case object`?

Reply:
@marcinwadon typo -- thanks for catching that.